packages/sdk/server-ai/README.md

```diff
@@ -0,0 +1,41 @@
+# LaunchDarkly Server-side AI library for Python
```

Comment:
Same here, we have just duplicated this file but left the original at the root. This make sense to move, but we should change the root README in that case.